lib/internal/bootstrap/loaders.js

```diff
@@ -155,6 +155,9 @@ function NativeModule(id) {
   this.loaded = false;
   this.loading = false;
   this.canBeRequiredByUsers = !id.startsWith('internal/');
+  // When using --expose-internals, we do not want to reflect the named exports
+  // from core modules as this can trigger getters.
+  this.defaultOnly = false;
```

Comment:
Can you state when this is true or when this is false? Or rename it to something more descriptive like `isForcedlyExposedInternals`?

Also I think this can just be computed out of `id.startsWith('internal/') && this.canBeRequiredByUsers` on the fly in `NativeModule.prototype.compileForPublicLoader` if we don't special case this for `loaders.js`?

Reply:
Sure I've replaced it with an inline `id.startsWith('internal/')` check.